source/triggers/database-triggers.txt

```diff
@@ -197,8 +197,7 @@ options are available:
        from immediately *before* the change was applied in the
        ``fullDocumentBeforeChange`` field. All change events except for
        **Insert** events include the document preimage. Document preimages are 
-       supported on non-sharded Atlas clusters running MongoDB 4.4+, and 
```

Comment:
why was this line removed? isn't it still true that it's supported on non-sharded clusters in 4.4+?

Reply:
@cbullinger according to the ticket, it is now supported on both sharded and unsharded running 5.3+:

"This (limitation) should be removed for Triggers on clusters running mongo version >=5.3."

Reply:
I mean, I guess I could just remove the entire sentence...